Apache-Beam-Jobs.md

```diff
@@ -49,26 +49,20 @@ If you're already familiar with Apache Beam or are eager to start writing a new
 
 Conceptually, an Apache Beam job is just a bunch of steps, each of which transforms some input data into some output data. For example, if you wanted to count how many interactions are in all of Oppia's explorations, you could break that task down into a series of transformations:
 
-    .--------------. Count interactions .--------. Sum .-------.
-    | Explorations | -----------------> | Counts | --> | Total |
-    '--------------'                    '--------'     '-------'
+```mermaid
+flowchart LR
+    A(Explorations) -->|Count interactions| B(Counts) -->|Sum| C(Total)
+```
 
 For more complicated tasks, Apache Beam supports tasks whose transformations form a [directed acyclic graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph), or "DAG." These are just graphs with no cycles. For example, if you wanted to find the ratio of interactions to cards, you could use this DAG:
-
-    .--------------. Count interactions .-------. Sum .------------------.
-    | Explorations | -----------------> | Count | --> | Num Interactions |
-    '--------------'                    '-------'     '------------------'
-           |                                                    |
-           |                                                    |
-           |                                                    |
-           | Count cards .-------. Sum .-----------.            |
-           '-----------> | Count | --> | Num Cards |------------+
-                         '-------'     '-----------'            |
-                                                                |
-                                .----------------------. Divide |
-                                | Interactions / Cards | <------'
-                                '----------------------'
-
+```mermaid
+flowchart TD
+E(Explorations) -->|Count interactions| C(Count) -->|Sum| NI(Num Interactions)
+E -->|Count cards| C2(Count) -->|Sum| NC(Num Cards)
+NC --> +(+)
+NI --> +
```

Comment:
Can we remove the + and directly connect it to Interactions / Cards?

Reply:
@gp201 -  Currently, it looks like this:
![one](https://user-images.githubusercontent.com/60394916/218565576-68680066-2e40-4e6e-a46b-58870ae654ed.PNG)

After removing the plus, it'll look like this:
![two](https://user-images.githubusercontent.com/60394916/218565719-d510c964-e1b2-49f1-a25c-386755fff3e7.PNG)

Should I still remove the plus?

Reply:
yep that works!

Reply:
@gp201 I have removed the +